x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -324,7 +323,7 @@ const UnstyledProcessEventDot = React.memo(
         >
           <g>
             <use
-              xlinkHref={`#${SymbolIds.processCubeActiveBacking}`}
+              xlinkHref={`#${symbolIDs(resolverComponentInstanceID).processCubeActiveBacking}`}
```

Comment:
:grey_question: Something isn't right here, this more or less defeats the purpose of using `<symbol>`s